### DISCUSSION TITLE
Community Tools Brainstorm

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/BountyBot#197] [General]
- Small group randomizer
- Urgency motivation for bounties: post a bounty today to get X bonus
- Urgency motivation: defeat the server boss before X
- Access server boss UI either through command or in scoreboard channel
- "Disable server bosses" in server config, default on, potentially unlocked by server level (config is difficulty coefficient on historic activity, 0 for off)

**Comment by SillySalamndr** [comment]
> Small group randomizer

This seems _promising_ toward the overall goals of the bot, but I don't think I have enough context to know what this is for.
